### ISSUE TITLE
R: model to ONNX

### ISSUE THREAD

**Issue by VladPerervenko** [dmlc/xgboost#8857]
Is it possible to add a model converter to ONNX ?

**Comment by trivialfis** [comment]
I think parsing and converting the JSON model (or equivalently, the ubjson model) would be a good starting point. Please let me know if you would like to work on it. ;-)

**Comment by VladPerervenko** [comment]
I agree, you can start with json after that try to convert JSON in ONNX.
I need to convert the model to ONNX for further use in production. If you offer a converter to JSON, I am ready to participate in testing it.
Good luck
